README.md

```diff
@@ -16,6 +17,134 @@ It lets your React components read data from a answers-headless, and dispatch ac
 </div>
 <br>
 
-## License
+## Installation
+
+```shell
+npm install @yext/answers-headless-react
+```
+
+## Getting Started - `AnswersHeadlessProvider`
+
+First, wrap your components in an AnswersHeadlessProvider.
+
+```tsx
+import { AnswersHeadlessProvider } from '@yext/answers-headless-react';
+import SearchBar from './SearchBar';
+import MostRecentSearch from './MostRecentSearch';
+import UniversalResults from './UniversalResults';
+
+function MyApp() {
+  return (
+    <AnswersHeadlessProvider
+      apiKey='your api key'
+      experienceKey='your experience key'
+      locale='en'
+    >
+      {/* Add components that use Answers as children */}
+      <SearchBar/>
+      <MostRecentSearch/>
+      <UniversalResults/>
+    </AnswersHeadlessProvider>
+  )
+}
+```
+
```

Comment:
Small nit: Include a minor blurb here about what this `Provider` gives them access to. That sets up the next two sections well.